org.gmusicbrowser.gmusicbrowser.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app-id": "org.gmusicbrowser.gmusicbrowser",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "command": "gmusicbrowser",
+    "finish-args": [
+        "--share=ipc", 
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=xdg-music",
+        "--socket=pulseaudio",
+        "--socket=session-bus",
+        "--env=PERL5LIB=/app/lib"
+    ],
+    "build-options" : {
+        "env": {"PERL5LIB": "/app/lib"}
+    },
+    "modules": [
+        "audio-libs.json",
```

Comment:
Any reason for splitting those into a new file?

Reply:
Not any particular reason, just for reusability and to make it a bit cleaner